grafana/grafana.libsonnet

```diff
@@ -5,7 +5,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
     namespace: 'default',
 
     versions+:: {
-      grafana: '5.2.4',
+      grafana: '5.4.3',
```

Comment:
We can straight go to `6.0.1` if we touch this right now.
The downstream kube-prometheus project already uses that version.

Reply:
Using that version now.